build.gradle.kts

```diff
@@ -0,0 +1,116 @@
+plugins {
+    idea
+    `java-library`
+    `maven-publish`
+    id("net.kyori.indra") version "3.1.3"
+    id("net.kyori.indra.git") version "3.1.3"
+    id("net.kyori.indra.publishing") version "3.1.3"
+    id("io.freefair.lombok") version "8.4"
+}
+
+indra {
+    github("GeyserMC", "MCProtocolLib")
+
+    mitLicense()
+    publishReleasesTo("opencollab-release-repo", "https://repo.opencollab.dev/maven-releases/")
+    publishSnapshotsTo("opencollab-snapshot-repo", "https://repo.opencollab.dev/maven-snapshots/")
+
+    configurePublications {
+        pom {
+            name = "MCProtocolLib"
+            url = "https://github.com/GeyserMC/MCProtocolLib/"
+            organization {
+                name = "GeyserMC"
+                url = "https://github.com/GeyserMC"
+            }
+            developers {
+                developer {
+                    id = "steveice10"
+                    name = "Steveice10"
+                    email = "Steveice10@gmail.com"
+                }
+                developer {
+                    id = "GeyserMC"
+                    name = "GeyserMC"
+                    url = "https://geysermc.org/"
+                }
+            }
+        }
+
+        versionMapping {
+            usage(Usage.JAVA_API) { fromResolutionOf(JavaPlugin.RUNTIME_CLASSPATH_CONFIGURATION_NAME) }
+            usage(Usage.JAVA_RUNTIME) { fromResolutionResult() }
+        }
+    }
+
+    javaVersions {
+        target(17)
+        strictVersions()
+        testWith(17)
+        minimumToolchain(17)
+    }
+}
+
+val repoName = if (version.toString().endsWith("SNAPSHOT")) "maven-snapshots" else "maven-releases"
+publishing {
+    repositories {
+        maven("https://repo.opencollab.dev/${repoName}/") {
+            credentials.username = System.getenv("OPENCOLLAB_USERNAME")
+            credentials.password = System.getenv("OPENCOLLAB_PASSWORD")
+            name = "opencollab"
+        }
+    }
+}
+
+dependencies {
+    // Minecraft related libraries
```

Comment:
Please use gradle version catalogs for dependencies.

Reply:
Alrighty. Will do. I usually see them more fit for multi-module projects, but igg there is a inofficial standard.